src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -97,18 +100,78 @@
     <Compile Include="..\..\src\Microsoft\Data\SqlClient\AssemblyRef.cs">
       <Link>Microsoft\Data\SqlClient\AssemblyRef.cs</Link>
     </Compile>
-    <Compile Include="..\..\src\Microsoft\Data\SqlClient\ActiveDirectoryAuthenticationProvider.cs">
-      <Link>Microsoft\Data\SqlClient\ActiveDirectoryAuthenticationProvider.cs</Link>
+    <Compile Include="..\..\src\Microsoft\Data\SqlClient\ColumnEncryptionKeyInfo.cs">
+      <Link>Microsoft\Data\SqlClient\ColumnEncryptionKeyInfo.cs</Link>
+    </Compile>
+    <Compile Include="..\..\src\Microsoft\Data\SqlClient\DataClassification\SensitivityClassification.cs">
+      <Link>Microsoft\Data\SqlClient\DataClassification\SensitivityClassification.cs</Link>
+    </Compile>
+    <Compile Include="..\..\src\Microsoft\Data\SqlClient\EnclaveDelegate.cs">
+      <Link>Microsoft\Data\SqlClient\EnclaveDelegate.cs</Link>
+    </Compile>
+    <Compile Include="..\..\src\Microsoft\Data\SqlClient\EnclavePackage.cs">
+      <Link>Microsoft\Data\SqlClient\EnclavePackage.cs</Link>
     </Compile>
```

Comment:
This file is in different 'AnyOS' group in main branch https://github.com/dotnet/SqlClient/blob/43296f98a56016f3faab7c5b2836887cde2e951b/src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj#L560

Reply:
They're in the same group:
 - [`<ItemGroup Condition="'$(OSGroup)' != 'AnyOS'">`](https://github.com/dotnet/SqlClient/blob/43296f98a56016f3faab7c5b2836887cde2e951b/src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj#L519) in main branch
 - `<ItemGroup Condition="'$(OSGroup)' != 'AnyOS'">` here at line 36

Could you link the item groups you're looking at?

Reply:
I expect it at line 538 in this file.